File: Łatwe/Java/Liczby Pierwsze/LiczbyPierwsze.java
import java.util.Scanner;

class LiczbyPierwsze {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for (int i = 0; i < n; i++) {
            int number = in.nextInt();
            if (isPrime(number) && number != 1)
                System.out.println("TAK");
            else
                System.out.println("NIE");
            }
        }

    public static boolean isPrime(int number) {
        for (int i = 2; i < number; i++) {
            if (number % i == 0)
                return false;
        }
        return true;
    }
}

File: Łatwe/C++/Liczby Pierwsze/LiczbyPierwsze.cpp
#include <iostream>

bool isPrime(int number);

int main() {
    int n;
    std::cin >> n;
    for (int i = 0; i < n; i++) {
        int number;
        std::cin >> number;
        if (isPrime(number) && number != 1)
            std::cout << "TAK\n";
        else
            std::cout << "NIE\n";
    }
}

bool isPrime(int number) {
    for (int i = 2; i < number; i++) {
        if (number % i == 0)
            return false;
    }
    return true;
}
